CollapseLauncher/XAMLs/MainApp/Pages/Dialogs/SimpleDialogs.cs

```diff
@@ -823,10 +825,11 @@ public static async Task<ContentDialogResult> Dialog_SteamShortcutCreationSucces
             if (play)
             {
                 panel.Children.Add(new TextBlock { Text = Lang._Dialogs.SteamShortcutCreationSuccessSubtitle3, TextWrapping = TextWrapping.Wrap, Margin = new Thickness(0, 2, 0, 2) });
+                panel.Children.Add(new TextBlock { Text = Lang._Dialogs.SteamShortcutCreationSuccessSubtitle7, TextWrapping = TextWrapping.Wrap, Margin = new Thickness(0, 2, 0, 2) });
```

Comment:
Preferably use WrapWholeWords instead of simple Wrap to eliminate the possibility of words cut in half

Docs: https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.textwrapping?view=winrt-22621